Packages/com.unity.inputsystem/InputSystem/Editor/PropertyDrawers/AssetSearchProviders.cs

```diff
@@ -0,0 +1,89 @@
+#if UNITY_EDITOR && UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Search;
+using UnityEditor.VersionControl;
+
+namespace UnityEngine.InputSystem.Editor
+{
+    internal static class SearchConstants
+    {
+        // SearchFlags : these flags are used to customize how search is performed and how search
+        // results are displayed.
+        internal static readonly SearchFlags SearchFlags = SearchFlags.Sorted |
+            SearchFlags.OpenPicker |
+            SearchFlags.Packages;
```

Comment:
I'd suggest it shouldn't have packages on by default. 
Turning this off makes it smaller and more manageable and more like the previous version

Reply:
(This also excludes the unwanted project wide input actions 'template' until its renamed)

Reply:
Makes sense, will update

Reply:
Resolving this since updated according to request